garage/tf/algos/ddpg.py

```diff
@@ -197,7 +197,7 @@ def train_once(self, itr, paths):
         if itr % self.n_epoch_cycles == 0:
             logger.log("Training finished")
             logger.log("Saving snapshot #{}".format(int(epoch)))
-            params = self.get_itr_snapshot(epoch, paths)
+            params = self.get_itr_snapshot(epoch)
```

Comment:
is `params` even used?

Reply:
No. And the `save_itr_params` is not necessary. I will delete it.